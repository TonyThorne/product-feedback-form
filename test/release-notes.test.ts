import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/vue'
import { describe, expect, it } from 'vitest'

import TestForm from '../src/pages/release-notes.vue'

describe('The release notes', () => {
  it('should render the release notes title', () => {
    render(TestForm)
    const header = screen.queryByRole('heading', { name: 'Release Notes' })
    expect(header).toBeInTheDocument()
    expect(header).toMatchSnapshot()
  })
})
