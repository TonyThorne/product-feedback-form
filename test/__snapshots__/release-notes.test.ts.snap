// Vitest Snapshot v1

exports[`The release notes > should render the release notes title 1`] = `
<h1>
  Release Notes
</h1>
`;
